packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/RectangularNodeStyle.java

```diff
@@ -114,13 +122,19 @@ public Builder borderStyle(LineStyle borderStyle) {
             return this;
         }
 
+        public Builder withHeader(boolean withHeader) {
+            this.withHeader = Objects.requireNonNull(withHeader);
```

Comment:
It a primitive `boolean`, this is not needed and will only cause an unnecessary boxing/unboxing.

Reply:
Done